docker/transformers-pytorch-gpu/Dockerfile

```diff
@@ -1,30 +1,25 @@
-FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04
+FROM nvidia/cuda:11.2.2-cudnn8-runtime-ubuntu20.04
```

Comment:
Think we need https://github.com/huggingface/transformers/blob/86119c115496ca773b82f8a1c8cbf2d4e44149fa/.github/workflows/self-scheduled.yml#L39 here as well no?

Reply:
Same, it's on line 21 ;-)